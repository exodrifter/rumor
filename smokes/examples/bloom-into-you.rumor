// Bloom Into You, Chapter 15: On Your Mark
yuu: Wah! Nanami-senpai!
touko: Yuu!
touko+ Oh, thank you for finding those for us!
yuu: No problem. Umm...
yuu+ Why are you coming in?
touko: Hmm?
