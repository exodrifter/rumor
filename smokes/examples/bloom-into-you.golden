yuu: "Wah!" <> " Nanami-senpai!"
touko: "Yuu!"
touko+ "Oh," <> " thank you for finding those for us!"
yuu: "No" <> " problem. Umm..."
yuu+ "Why" <> " are you coming in?"
touko: "Hmm?"